src/contracts/common/ExecutionEnvironment.sol

```diff
@@ -178,49 +178,41 @@ contract ExecutionEnvironment is Base {
         onlyAtlasEnvironment
         returns (SolverTracker memory)
     {
-        // bidValue is inverted; Lower bids are better; solver must withdraw <= bidAmount
-        if (solverTracker.invertsBidValue) {
-            // if invertsBidValue, record floor now
-            solverTracker.floor = solverTracker.etherIsBidToken
-                ? address(this).balance
-                : IERC20(solverOp.bidToken).balanceOf(address(this));
-        }
+        // Record the initial balance before postSolverCall
+        uint256 initialBalance =
+            solverTracker.etherIsBidToken ? address(this).balance : IERC20(solverOp.bidToken).balanceOf(address(this));
```

Comment:
(in case this PR is used: `initialBalance` doesn't seem to be used)

Reply:
Now fixed to record the final bid amounts before the postSolver call for both invertsBidValue and the normal case

Reply:
This would implement correctly this statement from Alex here ```for the normal bids we wanted to be able to move inventory out that we have received from the solver after the solver call.```  https://github.com/FastLane-Labs/atlas/pull/225#discussion_r1645406449

Assuming that we want to move the inventory out during the postSolver call and not during the allocateValue or postOps call instead.